person.rb

```diff
@@ -4,6 +4,9 @@ class Person
   attr_writer :id
   attr_accessor :name, :age
 
+  has_many :rentals
+  has_many :books, through: :rentals
+
   def initialize(age, _name: 'Unknown', _parent_permission: true)
```

Comment:
- [ ] Rentals should also be initialized. Please refer to this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md) for more information on how to do that.
- [ ] A method for adding rentals should be also present. You can read more on how to do that [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md)